src/containers/reports/crisis/CrisisReportSagas.js

```diff
@@ -540,101 +541,124 @@ function* getChargeEventsWatcher() :Generator<any, any, any> {
   yield takeLatest(GET_CHARGE_EVENTS, getChargeEventsWorker);
 }
 
-function* getCrisisReportV2Worker(action :SequenceAction) :Generator<any, any, any> {
-  const response = {};
-  try {
-    const {
-      value: {
-        reportEKID,
-        reportFQN,
-        reviewSchema,
-      }
-    } = action;
-
-    if (!isValidUUID(reportEKID) || !FQN.isValid(reportFQN)) throw ERR_ACTION_VALUE_TYPE;
-    yield put(getCrisisReportV2.request(action.id));
-
-    const app :Map = yield select((state) => state.get('app', Map()));
-    const reportESID = getESIDFromApp(app, reportFQN);
-
-    const reportRequest = call(
+function* getCrisisReportV2DataWorker(action :{ reportEKID :UUID, reportFQN :FQN, reportData ?:Map }) :Saga<Object> {
+  const {
+    reportEKID,
+    reportFQN = CRISIS_REPORT_CLINICIAN_FQN,
+    reportData
+  } = action;
+  if (!isValidUUID(reportEKID) || !FQN.isValid(reportFQN)) throw ERR_ACTION_VALUE_TYPE;
+  const app :Map = yield select((state) => state.get('app', Map()));
+  const reportESID = getESIDFromApp(app, reportFQN);
+
+  const neighborsResponse = yield call(
+    getReportsV2NeighborsWorker,
+    getReportsV2Neighbors({
+      reportEKIDs: [reportEKID],
+      reportFQN,
+    })
+  );
+
+  if (neighborsResponse.error) throw neighborsResponse.error;
+
+  const neighbors = neighborsResponse.data.get(reportEKID) || List();
+  const appTypeFqnsByIds = yield select((state) => state.getIn(['app', 'selectedOrgEntitySetIds']).flip());
+  const neighborsByFQN = groupNeighborsByFQNs(neighbors, appTypeFqnsByIds);
```

Comment:
this `groupNeighborsByFQNs` function sounds great and like it could be a good addition to `lattice-utils`!